programmers/짝수와-홀수.js

```diff
@@ -0,0 +1,10 @@
+function solution(num) {
+    var answer =''
```

Comment:
```suggestion
    let answer = ""
```
이렇게 변하는 변수 같은 경우는 let을 사용합니다